torch/csrc/jit/codegen/cuda/scheduler/pointwise_utils.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <torch/csrc/jit/codegen/cuda/compute_at_map.h>
+#include <torch/csrc/jit/codegen/cuda/ir_all_nodes.h>
+#include <torch/csrc/jit/codegen/cuda/ir_utils.h>
+#include <torch/csrc/jit/codegen/cuda/scheduler/utils.h>
+
+namespace torch {
+namespace jit {
+namespace fuser {
+namespace cuda {
+namespace pointwise_utils {
```

Comment:
I generally don't like to have too many namespaces. Do we need to have this as a separate namespace rather than having them all in `scheduler_utils`?

Reply:
I have no problem with using `scheduler_utils`, but looks like `reduction_utils.cpp` and `mma_utils.cpp` all have its own namespace. It seems that `scheduler_utils` is used for commonly useful things, and `reduction_scheduler_utils`, `mma_util` are used for things that are useful only to specific scheduler?

Reply:
It's just my general preference, but we already have some confusions among `ir_utils`, `lower_utils`, etc. If it's really specific to something, I agree that makes sense to have its own namespace, but otherwise, I think it'd just take more mental burden to remember which namespace has which methods. You never know something that appeared to be just useful for some specific thing can also be used for other cases.

Reply:
Agreed, but we're already following this convention for now, so we should take this as is and consider cleanup later. Utility files are a bit annoying to manage, but I actually like them being fairly dedicated 1:1 with the schedulers they're working with. It's hard for me to think about how we should more generically clean up these utility functions. Though I would say we more likely could go cleanup `ir_utils`, `codegen/utils`, and `lower_utils`.